web/libraries/pdf.js/web/viewer.js

```diff
@@ -1779,7 +1779,7 @@ const PDFViewerApplication = {
 exports.PDFViewerApplication = PDFViewerApplication;
 let validateFileURL;
 {
-  const HOSTED_VIEWER_ORIGINS = ["null", "http://mozilla.github.io", "https://mozilla.github.io"];
+  const HOSTED_VIEWER_ORIGINS = ["null", "http://mozilla.github.io", "https://mozilla.github.io","https://media.namiblii.org/","s3://namiblii-media/"];
```

Comment:
This should make it work for all LIIs, since what is being checked here is the who is hosting the viewer, not who is hosting the content:

```suggestion
  const HOSTED_VIEWER_ORIGINS = ["null", "http://mozilla.github.io", "https://mozilla.github.io", window.location.protocol + "//" + window.location.host];
```